src/psyclone/tests/test_files/dynamo0p3/20.1.2_cma_apply_disc.f90

```diff
@@ -2,8 +2,6 @@
 ! BSD 3-Clause License
 !
 ! Copyright (c) 2017, Science and Technology Facilities Council
-! (c) The copyright relating to this work is owned jointly by the Crown,
-! Met Office and NERC 2016.
 ! However, it has been created with the help of the GungHo Consortium,
```

Comment:
Please delete the three GungHo Consortium lines

Reply:
Deleted the reference to GungHo consortium.